### ISSUE TITLE
Skip alarms more than once

### ISSUE THREAD

**Issue by mentonin** [vicolo-dev/chrono#318]
**Is your feature request related to a problem? Please describe.**
Skipping the next alarm is useful, but sometimes I want to skip them for a few days, e.g. when on vacation.

**Describe the solution you'd like**
An option similar to skip next alarm, but that lets you choose when to start alarm again, with a date and time picker.

**Describe alternatives you've considered**
Disabling the alarms and manually re-enabling later works, but requires remembering to re-enable them.

**Comment by KeronCyst** [comment]
Yes, Google Clock can do this and Chrono currently can't: temporarily pause an alarm for an entire range of specified days. I would appreciate and use this feature, too.
